WebColock.Tests/ClockControllerTests.cs

```diff
@@ -27,10 +28,17 @@ public void GetCurrentTime_WhenCalled_ReturnString()
         public void GetCurrentTime_WhenCalled_ReturnCorrectSecond()
         {
             var result = _controller.GetCurrentTime();
-
             var currentDate = JsonConvert.DeserializeObject<Clock>(result);
 
-            Assert.InRange(DateTime.Now.Second, currentDate.Second - 15, currentDate.Second + 15);
+            DateTime dt = new DateTime(
+                DateTime.Now.Year,
```

Comment:
Tu mamy niebezpieczny zapis z wielokrotnym pobieraniem (różnego) Now, wykorzystanym do inicjalizacji jednego obiektu. 
Zwykle to będzie dobrze, ale załóżmy, że trzy kolejne pobrania zwrócą:
- 2020-12-12 23:59:59,
- 2021-01-01 00:00:00, 
- 2021-01-01 00:00:00,
wtedy skonstruujemy datę 2020-01-01, która może się różnić więcej niż 15 sekund od currentDate

Reply:
Racja, poprawione.